src/main/java/net/atos/zac/mail/MailService.java

```diff
@@ -0,0 +1,122 @@
+    private RESTInformatieobjecttypeConverter restInformatieobjecttypeConverter;
+
+    @Inject
+    @IngelogdeMedewerker
+    private Instance<Medewerker> ingelogdeMedewerker;
+
+    private final ClientOptions clientOptions = ClientOptions.builder().apiKey(MAILJET_API_KEY).apiSecretKey(MAILJET_API_SECRET_KEY).build();
+    private final MailjetClient mailjetClient = new MailjetClient(clientOptions);
+
+    public int sendMail(final String ontvanger, final String onderwerp, final String body, final UUID zaakUuid) throws MailjetException {
+        final EMail eMail = new EMail(body, new Verstuurder(), List.of(new Ontvanger(ontvanger)), onderwerp);
+
+        final MailjetRequest request = new MailjetRequest(Emailv31.resource)
+                .setBody(JsonbBuilder.create().toJson(new EMails(List.of(eMail))));
+
+        final MailjetResponse response = mailjetClient.post(request);
+
+        if (response.getStatus() == Response.Status.OK.getStatusCode()) {
+            createAndSaveDocumentFromMail(body, onderwerp, zaakUuid);
+        }
+
+        return response.getStatus();
+    }
+
+    private void createAndSaveDocumentFromMail(final String body, final String onderwerp, final UUID zaakUuid) {
+        final Zaak zaak = zrcClientService.readZaak(zaakUuid);
+        final EnkelvoudigInformatieobjectWithInhoud data = createDocumentInformatieObject(zaak, onderwerp, body);
+        zgwApiService.createZaakInformatieobjectForZaak(zaak, data, onderwerp, onderwerp, OMSCHRIJVING_VOORWAARDEN_GEBRUIKSRECHTEN);
+    }
+
+    private EnkelvoudigInformatieobjectWithInhoud createDocumentInformatieObject(final Zaak zaak,
+            final String onderwerp, final String body) {
+        final RESTInformatieobjecttype eMailObjectType =
+                listEnkelvoudigInformatieobjectenVoorZaak(zaak).stream()
+                        .filter(objectType -> objectType.omschrijving.equals("e-mail")).findFirst().orElseThrow();
+
+        final EnkelvoudigInformatieobjectWithInhoud data = new EnkelvoudigInformatieobjectWithInhoud(
+                ConfigurationService.BRON_ORGANISATIE, LocalDate.now(), onderwerp, ingelogdeMedewerker.get().getNaam(),
+                "NLD", eMailObjectType.url, Base64.getEncoder().encodeToString(body.getBytes(StandardCharsets.UTF_8)));
+        data.setVertrouwelijkheidaanduiding(Vertrouwelijkheidaanduiding.OPENBAAR);
+        data.setFormaat("text/plain");
+        data.setBestandsnaam(String.format("%s.txt", onderwerp));
+        data.setStatus(InformatieobjectStatus.DEFINITIEF);
+        data.setIndicatieGebruiksrecht(false);
+        data.setVertrouwelijkheidaanduiding(Vertrouwelijkheidaanduiding.OPENBAAR);
+
+        return data;
+    }
+
+    private List<RESTInformatieobjecttype> listEnkelvoudigInformatieobjectenVoorZaak(final Zaak zaak) {
+        final Zaaktype zaaktype = ztcClientService.readZaaktype(zaak.getZaaktype());
+        return restInformatieobjecttypeConverter.convert(zaaktype.getInformatieobjecttypen());
```

Comment:
Het is niet nodig om de Informatieobjecttypen te converteren naar een REST object. Hier gewoon een call doen naar ztcClientService::readInformatieobjecttype en Informatieobjecttype gebruiken. Kijk maar wat er gebeurt in de converter.